src/pages/admin/reports/components/block-component/ReportsMediaBlock.module.scss

```
@@ -75,8 +84,13 @@
         1;
 }
 
+.description {
+    color: c.$grey-900;
+}
+
 .title {
     @include t.subtitle-1-medium;
+    font-weight: 700;
```

Stylelint (17.5.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)